px-k8s-sql-ha/wait4-k8s-px.md

```diff
@@ -0,0 +1,18 @@
+Let's first wait for Kubernetes and Portworx to be ready.
+
+### Step: Wait for Kubernetes to be ready
+
+Click the below section which waits for all Kubernetes nodes and Portworx to be ready.
+```
+./wait4-k8s-px.sh
+```{{execute T1}}
+
+And let's make sure `Stork` is enabled for scheduler convergence.
```

Comment:
This is no longer needed as latest install.portworx.com now already deploys stork.